java/core/src/main/java/sleeper/core/partition/PartitionTree.java

```diff
@@ -159,6 +160,54 @@ public Partition getNearestCommonAncestor(Partition a, Partition b) {
         return getRootPartition();
     }
 
+    public Stream<Partition> traverseLeavesFirst() {
```

Comment:
Can you add some Javadoc to say what this method does?

Reply:
Will do, thanks.

Reply:
I've added that.